src/main/java/com/smoothstack/december/orchestrationservice/controller/LibrarianController.java

```diff
@@ -25,24 +23,14 @@
 
     private static final String baseUrl = "http://localhost:8081/lms/librarian";
```

Comment:
I made my base URL /v1/lms/borrower-service, trying to find follow the guidelines and examples I've seen. I don't know if I'm following them correctly...but either way, we should be consistent.